### PR TITLE
RHOAIENG-15152: test(tests/browser): add playwright test for starting code-server

### PR DESCRIPTION
**Pull request by jiridanek**
https://issues.redhat.com/browse/RHOAIENG-15152

## Description

Followup to

* https://github.com/opendatahub-io/notebooks/pull/755

## How Has This Been Tested?

Played with it on my machine, and let the GHA run, see the attached test report Artifacts at
* https://github.com/opendatahub-io/notebooks/actions/runs/11880669927?pr=774

(before I reverted the whitespace change I used to force image rebuild and retest).

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: tests/browser/tests/utlis.ts
Comment (andyatmiami) [inline tests/browser/tests/utlis.ts:1]: typo i think... presumably we want the file to be named `utils.ts`

**Comment by jstourac** [comment]
I don't say I understand this fully, but LGTM in general. Thank you!

/lgtm

**Pull request merged**